src/content/posts/releases/mitmproxy-11/index.md

```diff
@@ -0,0 +1,164 @@
+
+<!--more-->
+
+##### *Editorial Note:*
+
+*Hi! I'm [Gaurav Jain], one of the students selected for this year's Google Summer of Code program to work on mitmproxy.
+During this summer, I've worked on improving various low-level networking parts of mitmproxy some of which include
+HTTP/3 and DNS. You can find my project report [here].*
+
+[Gaurav Jain]:  {{< relref "/authors/gaurav-jain" >}}
+[here]: https://gist.github.com/errorxyz/af6f26549e9122f3ff3b93fd9d257df1
+
+## HTTP/3
+
+HTTP/3 now "just works" for reverse proxies. Your mitmproxy instance will listen for
+both TCP and UDP packets and handle all HTTP versions thrown at it:
+
+```shell
+$ mitmproxy --mode reverse:https://http3.is
+```
+
+Our transparent proxy modes now all support HTTP/3 as well:
+
+```shell
+$ mitmproxy --mode wireguard
+$ mitmproxy --mode local
+$ mitmproxy --mode transparent
+```
+
+We have successfully tested HTTP/3 support with Firefox, Chrome, various cURL builds, and other clients to iron out 
+compatibility issues.
+The only limitation we are aware of is that Chrome [does not trust user-added Certificate Authorities for QUIC].
+This means your will either need to provide a publicly trusted certificate (e.g. from Let's Encrypt), start Chrome with 
+a [command line switch], or accept that it falls 
+back to HTTP/2. Alternatively, Firefox doesn't do such shenanigans.
+
+Bringing HTTP/3 support to mitmproxy is a major effort that was started in 2022 by [Manuel Meitinger] and [Maximilian Hils]. 
+QUIC and HTTP/3 make up an increasing share of network traffic in the wild, and we're super excited to have this ready
+and enabled by default now!
+
+
+
+[Manuel Meitinger]: {{< relref "/authors/manuel-meitinger" >}}
+[Maximilian Hils]:  {{< relref "/authors/maximilian-hils" >}}
+[initial codebase]: {{< relref "/posts/releases/mitmproxy-10.1" >}}
+[does not trust user-added Certificate Authorities for QUIC]: https://issues.chromium.org/issues/40138351#comment15
+[command line switch]: https://www.chromium.org/quic/playing-with-quic/#generate-certificates
+[easily disabled]: https://docs.mitmproxy.org/stable/concepts-options/#http3
+
+
+<!--
+In the background, mitmproxy automatically updates [Alt-Svc] headers and strips [Encrypted Client Hello]
+records,
+Note that we can run the reverse proxy on any port irrespective of the port advertised by the [alt-svc] header.
+We also support proxying HTTP3 flows via local-redirect mode.
+
+[Alt-Svc]: https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Alt-Svc
+[Encrypted Client Hello]: https://blog.cloudflare.com/announcing-encrypted-client-hello/
+
+```shell
+$ mitmproxy --mode local --set experimental_transparent_http3=true
+```
+
+Any cURL, Chromium or Firefox based HTTP/3 client ([setup]) can be used to access these features. There are still some
+bugs that are yet to be fixed which you can find in our [bug-tracker].
+
+[setup]: https://github.com/mitmproxy/mitmproxy/issues/7025#issuecomment-2351138170
```

Comment:
I think we should mention this comment somewhere - it'd save a lot of debugging time :)